model/article-model.js

```diff
@@ -10,4 +10,12 @@ exports.selectArticleById = (id) => {
       return rows[0]
     })
     
+}
+
+exports.selectAllArticles = () => {
+let queryString = "SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url ORDER BY articles.created_at DESC;"
+return db.query(queryString).then(({ rows }) => {
+  
+  return rows
+})
 }
```

Comment:
Looks good, you cast `cast` the comment count as an integer if you want

Reply:
Thank you thats awesome ive introduced CAST and reflected that in the tests appreciate the direction